src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -334,6 +335,19 @@ public boolean hasPermission(Rank rank, @NotNull Action action)
     }
 
     /**
+     * Checks if the player has the permission of an action
+     *
+     * @param player {@link EntityPlayer} player
+     * @param action {@link Permissions.Action} action
+     * @return true if player has permissions, otherwise false
+     */
+    @Override
+    public boolean hasPermission(@NotNull EntityPlayer player, @NotNull Action action)
+    {
+        return hasPermission(getRank(player), action);
+    }    
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ATrailingCommentCheck)